src/regtech_cleanup_api/entities/repos/filing_repo.py

```diff
@@ -16,16 +16,24 @@
 T = TypeVar("T")
 
 
-def get_user_action_ids(session: Session, lei: str = None, period_code: str = None):
-    filing = submission_repo.get_filing(session, lei, period_code)
+def get_user_action_ids(
+    session: Session,
+    lei: str = None,
+    period_code: str = None,
+    submissions: bool = False,
+):
+    filing_user_action_id = []
+    if not submissions:
+        filing = submission_repo.get_filing(session, lei, period_code)
+        filing_user_action_id = (
+            [filing.creator_id] if filing.creator_id is not None else []
+        )
     submissions = submission_repo.get_submissions(session, lei, period_code)
```

Comment:
A little confused by this.  Should there be an else?  Is the submissions boolean supposed to only add submission useraction ids if it's true?

Reply:
Yes, this was changed to gather filing and submissions by default OR just submissions based on the flag. Default for the remove all endpoint, the flag set to True for the remove submissions endpoint.